tutorials/W1D2_StateoftheClimateOceanandAtmosphereReanalysis/W1D2_Tutorial3.ipynb

```diff
@@ -508,15 +513,13 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "execution": {}
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# to_remove explanation\n",
     "'''\n",
     "1. The Hadley Cell is the tropical region containing the easterlies (a.k.a. trade winds), and the Ferrel Cell is the mid-latitude region containing the westerlies.\n",
-    "2. Meridional winds are typically weaker and less variable than zonal winds. This is due to the general circulation of the atmosphere, which primarily moves air in the east-west direction rather than north-south.\n",
+    "2. Meridional winds are typically weaker and less variable than zonal winds. This is due to the general circulation of the atmosphere, which primarily moves air in the east-west direction rather than north-south.TODO add smt here regarding longtiude and latitude\n",
     "''';"
    ]
   },
```

Comment:
Do you want to add something before we merge the changes?

Reply:
Will remove this comment for now, will think about it and wait for feedback from the pre-pod.

Reply:
Most changes in W2D4 were made intentionally but on the wrong branch. Hope that works out... Sorry, my bad.